### DISCUSSION TITLE
Add 'Blog'  feature om Bagisto CMS

### DISCUSSION THREAD

**Discussion by abusayed0206** [bagisto/bagisto#6562] [Ideas]
I didn’t find any simple way to add blogpost on Bagisto. There only an option to publish cms page.

**Comment by Sanjaybhattwebkul** [comment]
Hi there, 
Please take a reference from [Here](https://devdocs.bagisto.com/1.x/packages/create-package.html) for the same.

**Comment by devansh-webkul** [comment]
Bagisto's primary feature is for commerce solutions. However, we also have a CMS page feature. If this still doesn't meet your requirements, you can refer to the link provided by Sanjay and create your own package.
